File: praktikum_6/PRAK601-2410817210004-Rahmad.c
#include <stdio.h>
int main(){
    int baris,kolom;
    scanf("%d",&baris);
    scanf("%d",&kolom);

    int matriks[baris][kolom];
    for(int i= 0;i<baris;i++){
        for(int j= 0;j<kolom;j++){
        scanf("%d",&matriks[i][j]);
        }
    }
    printf("\n");
    for(int i= 0;i<baris;i++){
        for(int j= 0;j<kolom;j++){
        printf("%d ",matriks[i][j]);
        }
        printf("\n");
    }
    
}

File: praktikum_6/PRAK601-2410817210004-Rahmad.py
baris=int(input())
kolom=int(input())

matriks=[]
bilangan = list(map(int, input("").split()))
index=0
for i in range(baris):
    baris_matriks = bilangan[index:index + kolom]
    matriks.append(baris_matriks)
    index+=kolom    
    ##for j in range(kolom):
    ##    elemen = int(input(" "))
    ##    baris_matriks.append(elemen)
    ##matriks.append(baris_matriks)
print("\n matriks yang diinput : ")
for row in matriks :
    print(row)
